examples/kubernetes/README.md

```diff
@@ -21,7 +21,7 @@ You'll need [Go 1.3+](http://golang.org/doc/install) in order to build the
 ### Build and install vtctlclient
 
 ```
-$ go get github.com/youtube/vitess/go/cmd/vtctlclient
+vitess/examples/kubernetes$ go build ../../go/cmd/vtctlclient
```

Comment:
Yep, as discussed offline it was just an issue of leftover files from previous calls to go get (prior to my earlier fix).  Reverted the change.